repose-aggregator/src/docs/asciidoc/architecture/system-model.adoc

```diff
@@ -170,6 +170,44 @@ If set to localhost, any physical node will match and run the *Repose* node.
      In this case, no services are being used.
 <13> Defines the required one default endpoint.
 
+==== Streaming Request Body to the Origin Service
+Streaming a request body to the origin service is accomplished by using the `chunked` transfer encoding.
+
+[source,xml]
+.partial http-connection-pool.cfg
+----
+<system-model xmlns="http://docs.openrepose.org/repose/system-model/v2.0">
+  <repose-cluster id="repose_cluster1"
+                  chunked-encoding="auto"> <!--1-->
+
+    <nodes>
+      <node id="repose_node", hostname="localhost", http-port="8080"/>
```

Comment:
lol, copied commas

Reply:
Dang it. I caught it in the `destination`, but not on the `node`, apparently. Will fix.